src/annotator/range-util.js

```diff
@@ -14,49 +14,43 @@ export function isSelectionBackwards(selection) {
 }
 
 /**
- * Returns true if `node` lies within a range.
- *
- * This is a simplified version of `Range.isPointInRange()` for compatibility
- * with IE.
+ * Returns true if any part of `node` lies within `range`.
  *
  * @param {Range} range
  * @param {Node} node
  */
 export function isNodeInRange(range, node) {
-  if (node === range.startContainer || node === range.endContainer) {
-    return true;
+  try {
```

Comment:
I note that MDN still marks `comparePoint` as "experimental." It's supported in the browsers we care about, but I thought I'd ask after that...

Edited: I see _your_ comment about support in Edge 17+ ... my question is broader now: what's our stance on "experimental" APIs, if any?

Reply:
The information on MDN is out of date as far as I can tell. The compatibility information on [caniuse](https://caniuse.com/mdn-api_range_comparepoint) is incomplete, but this appears to be because the API is so old. I found references to it in WebKit in 2008 and a Firefox bug report about some spec compliance details in 2013. An API that has existed in multiple major browsers for that long will never get breaking changes unless it causes a major security or performance problem and nobody is using it.

Our general policy has been that once an API (or ES language feature) ships on-by-default in the stable version of two major browsers it is safe to use. Special consideration might have to be given to very complex features (eg. Service Workers or Web Components).